robottelo/ui/rhai.py

```diff
@@ -27,23 +27,19 @@ def register_client_to_rhai(self, activation_key, org):
         self.vm.register_contenthost(activation_key, org)
 
         # Red Hat Access Insights requires RHEL 6/7 repo and it not
-        # possible to sync the repo during the tests,
-        # adding a file in /etc/yum.repos.d/rhel6/7.repo
+        # possible to sync the repo during the tests, Adding repo file.
 
-        rhel6_repo = conf.properties['insights.rhel6_repo']
+        rhel6_repo = conf.properties['clients.rhel6_repo']
+        self.vm.configure_rhel_repo(rhel6_repo)
```

Comment:
The `rhel6_repo` variable is only used once. I think it can reasonably be dropped.

```python
        self.vm.configure_rhel_repo(conf.properties['clients.rhel6_repo'])
```